### PR TITLE
Add line_spacing property and change (0,0) definition

### PR DESCRIPTION
**Pull request by ladyada**
the origin point of text areas is from the baseline of the font rather than the font's bounding box corner - that way changing fonts will not change the location of the text, only size and style

see this for definition of font baseline
https://printingpartners.files.wordpress.com/2017/12/font-diagram.png

see pygame which does the same (i think) https://nerdparadise.com/programming/pygame/part5

@tannewt its a breaking change so i'd like to do this soon if you're OK with it

### CODE REVIEW

**Comment by ladyada** [comment]
@tannewt this is the new version with center-M origin

File: adafruit_display_text/label.py [hunk 1]
```diff
@@ -44,21 +44,20 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Display_Text.git"
 
-class TextArea(displayio.Group):
+class Label(displayio.Group):
     """An area displaying a string of textself.
```
Comment (tannewt) [inline adafruit_display_text/label.py]: Please add a comment about how the text is positioned relative to the position.

**Comment by ladyada** [comment]
@tannewt i need to lint more

File: adafruit_display_text/label.py [hunk 2]
```diff
@@ -44,21 +44,23 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Display_Text.git"
 
-class TextArea(displayio.Group):
-    """An area displaying a string of textself.
+class Label(displayio.Group):
+    """An area displaying a string of textself. The origin point set by ``x`` and ``y``
```
Comment (tannewt) [inline adafruit_display_text/label.py]:
```suggestion
    """A label displaying a string of text. The origin point set by ``x`` and ``y``
```

**Pull request merged**